packages/launcher/test/unit/windows_spec.ts

```diff
@@ -87,6 +89,46 @@ describe('windows browser detection', () => {
     snapshot(await detect(firefoxes))
   })
 
+  it('works with :browserName format in Windows', () => {
+    sinon.stub(os, 'platform').returns('win32')
+    stubBrowser(`${HOMEDIR}/foo/bar/browser.exe`, 'foo-browser v100')
```

Comment:
This is actually stubbing the WMIC version output, not the `--version` output, so this should read:

```suggestion
    stubBrowser(`${HOMEDIR}/foo/bar/browser.exe`, '100')
```